templates/override.conf.j2

```diff
@@ -4,4 +4,8 @@ LimitMEMLOCK = infinity
 {% endif %}
 {% if es_max_open_files is defined %}
 LimitNOFILE = {{ es_max_open_files }}
-{% endif %}
\ No newline at end of file
+{% endif %}
+{% if sysd_always_restart %}
+Restart=always
```

Comment:
FYI systemd recommends using `Restart=on-failure` to do what we want. See [here](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Restart=). But that won't make any difference in the current situation :)